mobile/android/android-emulator-services.ts

```diff
@@ -32,6 +32,20 @@ class AndroidEmulatorServices implements Mobile.IEmulatorPlatformServices {
 		iconv.extendNodeEncodings();
 	}
 
+	public checkDependencies(): IFuture<void> {
+		var future = new Future<void>();
+		var proc = this.$childProcess.spawn('emulator', ['-help']);
+		proc.on("error", (args:any) => {
+			this.$errors.fail("The Android SDK is not configured properly. " +
+				"Verify that you have installed the Android SDK and that you have added its `platform-tools` and `tools` directories to your PATH environment variable.");
+		});
+		proc.on("exit", (code:number) => {
+			future.return();
```

Comment:
is it possible to have both error and exit events, we stick to the following code:
```Typescript
if (!future.isResolved()) { 
    future.return();
}
```
Same is valid for on error callback